generators/entity-server/index.js

```diff
@@ -201,15 +201,6 @@ module.exports = class extends BaseBlueprintGenerator {
   // Public API method used by the getter and also by Blueprints
   _writing() {
     return {
-      cleanupOldElasticSearchFiles() {
-        if (!this.searchEngineElasticsearch) return;
-        if (this.isJhipsterVersionLessThan('7.6.1')) {
-          this.removeFile(`${this.entityAbsoluteFolder}/repository/search/SortToFieldSortBuilderConverter.java`);
-        }
-        if (this.isJhipsterVersionLessThan('7.7.1')) {
-          this.removeFile(`${this.entityAbsoluteFolder}/repository/search/SortToSortBuilderListConverter.java`);
-        }
-      },
```

Comment:
The idea is to have a method for each technology for a future modularization.
IMO we should keep as it is now.

Reply:
I first wanted to extract the cleanup method to have everything at the same place, then split by tech. Please leave this one, I'll then make the split